docs/self-managed/concepts/multi-region/dual-region.md

```diff
@@ -0,0 +1,161 @@
+
+In an **active-active** setup, multiple application instances run simultaneously in different regions, actively handling user requests. This allows for better load balancing and fault tolerance, as traffic can spread across regions. If one region fails, the workload can shift to another without causing disruptions.
+
+In contrast, an **active-passive** setup designates one region as the main or active region where all user requests are processed. The other region remains on standby until needed, only becoming active if the primary region fails. This setup is easier to manage but may result in higher delays during failover events.
+
+### User traffic
+
+The overall system is **active-passive**, even though some components may be **active-active**. You will have to take care of the user traffic routing or DNS by yourself, and won't be considered further. You must select one region as the actively serving region and route the user traffic there. In case of a total region failure, you must route the traffic to the passive region yourself.
+
+<!-- Should we provide some reading materials on how to tackle this? -->
+
+### Components
+
+#### Zeebe ![active](https://img.shields.io/badge/active-green?style=for-the-badge)
+
+Zeebe, as a workflow engine, is fully **active-active** and replicates data between its brokers. Due to its replication logic, it can handle an entire region failure without data loss. Requiring a proper partition and replication setup. Consider reading through the [cluster technical concept](./../../../components/zeebe/technical-concepts/clustering.md) to learn more about the [raft protocol](<https://en.wikipedia.org/wiki/Raft_(algorithm)>).
+
+#### Elasticsearch ![passive](https://img.shields.io/badge/passive-orange?style=for-the-badge)
+
+We treat Elasticsearch as an **active-passive** component. While it may be possible to deploy it in active-active mode and stretch across regions, it's not officially documented by Elasticsearch. Such a setup generally brings its own challenges along that we can't control and may negatively impact the performance of Zeebe.
+
+We go with the approach of keeping an Elasticsearch per region and populating the data directly from the Zeebe exporter in each region.
+
+#### WebApps ![passive](https://img.shields.io/badge/passive-orange?style=for-the-badge)
+
+WebApps are an **active-passive** component and in its current state also not highly available since we're limited by the included exporters that would cause data issues when running multiple instances at the same time.
+
+Per region, one WebApp instance is required for the exporter to run.
+
+This means that one instance will be actively serving traffic, while the other one is on standby.
+
+##### Operate
+
+In the event of a total primary region loss, the following data will be lost:
```

Comment:
I think it's about a chance of losing it, not guaranteed a loss:

```suggestion
In the event of a total primary region loss, the following data can be lost:
```

Reply:
if your primary region is lost the data is lost since it's solely in that region.
That's why Thorben is working on moving that to the engine so the data is replicated as well.